apps/frontend-manage/src/components/courses/PointCorrectionsModal.tsx

```
@@ -0,0 +1,433 @@
+      participantScope: Yup.mixed<PointCorrectionType>()
+        .oneOf([
+          PointCorrectionType.Single,
+          PointCorrectionType.Participating,
+          PointCorrectionType.AllCourse,
+        ])
+        .required(),
+      participantId: Yup.string()
+        .trim()
+        .when('participantScope', {
+          is: 'single',
+          then: (schema) =>
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)